cw-orch-daemon/src/sender.rs

```diff
@@ -115,13 +144,21 @@ impl Sender<All> {
         Ok(self.pub_addr()?.to_string())
     }
 
+    pub fn message_sender(&self) -> Result<AccountId, DaemonError> {
```

Comment:
We should add a doc comment here, and wouldn't `msg_sender` be more consistent with use of `msg` elsewhere?